### ISSUE TITLE
add language model to demo

### ISSUE THREAD

**Issue by joelgrus** [allenai/allennlp-demo#181]
motivation: language models are hot. We already have [a demo for one](https://gpt2.apps.allenai.org/?text=Joel%20is) but it's not part of the AllenNLP demo. It would be make our demo more appealing / interesting if we had language modeling as part of it.

Language models are big, so we probably can't do this until after #180 

success criteria: have a language model demo (likely similar to the gtp2-explorer) which can be configured between at least three language models (such as GPT2, BERT, and ELMo--but maybe we'll have better examples from Irvine).  ELMo is a clear requirement.

**Comment by matt-gardner** [comment]
@rloganiv has some language model implementations that we'd like to get in here, also (an AWD-LSTM and an EntityNLM, to start with).  I think there's some work to do to fix up some APIs to make them better integrated with the library first, though.

**Comment by dnlbreen** [comment]
Not sure if this is an appropriate place to post this question, but I've found that question answering is very slow to evaluate with BERT. For answering about 20 questions across around 50 documents of 1000s of words, for example, it takes several hours to evaluate using a cpu. With a GPU, it still takes around 20-30 minutes. Even the smaller BIDAF model on your demo seems to spend a lot of time, around a minute, to evaluate questions on documents with 1000s of words. Do you know why this is? Are deep learning question answering models practical for real time question answering on documents?

**Comment by schmmd** [comment]
This is done. @dnlbreen I don't know how to answer your question--but you're right this probably isn't the best forum as it's focused on the AllenNLP demo.
